doc/ping.xml

```diff
@@ -964,8 +977,8 @@ xml:id="man.ping">
     executed 1) if the program is used for non-echo queries (See
     <option>-N</option> option), or 2) if kernel does not support
     ICMP datagram sockets, or 3) if the user is not allowed to
-    create an ICMP echo socket. The program may be used as set-uid
-    root.</para>
+    create an ICMP echo socket, or 4) uses raw sockets for any reason.
+    The program may be used as set-uid root.</para>
```

Comment:
Instead of `4) uses raw sockets for any reason.` you should change 1) to:

```
... 1) if the program is used for non-echo queries (See
<option>-N</option> option) or set identification field of the 
<emphasis remap="I">ECHO_REQUEST</emphasis> (See
 <option>-e</option>), ...
```

Reply:
done